tests/test_parsers.py

```diff
@@ -3,33 +3,22 @@
 """
 import gzip
 import shutil
+from io import StringIO
 
 import pytest
 
 from ralph.parsers import GELFParser
 
 
-def test_gelfparser_parse_non_existing_file():
-    """Test the GELFParser with a file path that does not exist."""
-
-    parser = GELFParser()
-
-    # Input log file does not exists
-    with pytest.raises(OSError):
-        next(parser.parse("/i/do/not/exist"))
-
-
 # pylint: disable=invalid-name
-def test_gelfparser_parse_empty_file(fs):
+def test_gelfparser_parse_empty_file():
     """Test the GELFParser parsing with an empty file."""
 
     parser = GELFParser()
 
-    # Input log file is empty
-    empty_file_path = "/var/log/empty"
-    fs.create_file(empty_file_path)
+    empty_file = StringIO()
     with pytest.raises(StopIteration):
-        next(parser.parse(empty_file_path))
+        next(parser.parse(empty_file))
```

Comment:
Removed the ability to parse files by specifying a path as the cli parse command only reads from standart input.
May be I could add an option to the parse command to specify a file instead of the standart input ? Then we could keep this tests unchanged?

Reply:
For now, we suppose to have a handle on a file.